.editorconfig

```diff
@@ -11,3 +11,9 @@ charset = utf-8
 trim_trailing_whitespace = true
 indent_style = space
```

Comment:
@YohannParis I forget what we decided... tabs or spaces?

Reply:
Tabs